packages/dapp/src/components/StoryFeed/StoryFeed.tsx

```diff
@@ -76,20 +76,28 @@ class StoryFeedPage extends React.Component {
                 return "Error loading Story Feed.";
               }
 
-              return feedQueryData.postsSearch.posts.map((storyData: any, i: number) => (
-                <StoryFeedItem
-                  key={i}
-                  storyId={storyData.id}
-                  activeChallenge={false}
-                  createdAt={storyData.createdAt}
-                  isStripeConnected={storyData.channel.isStripeConnected}
-                  newsroom={storyData.channel.newsroom}
-                  openGraphData={storyData.openGraphData}
-                  displayedContributors={storyData.payments}
-                  sortedContributors={storyData.payments}
-                  totalContributors={storyData.payments ? storyData.payments.length : 0}
-                />
-              ));
+              return feedQueryData.postsSearch.posts.map((storyData: any, i: number) => {
+                return (
+                  <>
+                    {storyData.openGraphData && storyData.openGraphData.title && storyData.openGraphData.url && (
```

Comment:
Require at minimum og title and url. This shouldn't be a problem for 99.9% of newsrooms but might come up. We should also let them know somewhere what og data we use.